cpp/src/barretenberg/stdlib/recursion/verifier/verifier.test.cpp

```diff
@@ -449,267 +440,163 @@ template <typename OuterComposer> class stdlib_verifier : public testing::Test {
 
         circuit_output_a.aggregation_state.assign_object_to_proof_outputs();
 
-        auto circuit_output_b = create_outer_circuit(inner_composer_b, mid_composer_b);
+        auto circuit_output_b = create_outer_circuit(inner_circuit_b, mid_circuit_b);
 
         circuit_output_b.aggregation_state.assign_object_to_proof_outputs();
 
-        auto circuit_output = create_double_outer_circuit(mid_composer_a, mid_composer_b, outer_composer);
-
+        auto circuit_output = create_double_outer_circuit(mid_circuit_a, mid_circuit_b, outer_circuit);
         circuit_output.aggregation_state.assign_object_to_proof_outputs();
-
-        g1::affine_element P[2];
-        P[0].x = barretenberg::fq(circuit_output.aggregation_state.P0.x.get_value().lo);
-        P[0].y = barretenberg::fq(circuit_output.aggregation_state.P0.y.get_value().lo);
-        P[1].x = barretenberg::fq(circuit_output.aggregation_state.P1.x.get_value().lo);
-        P[1].y = barretenberg::fq(circuit_output.aggregation_state.P1.y.get_value().lo);
-        barretenberg::fq12 inner_proof_result = barretenberg::pairing::reduced_ate_pairing_batch_precomputed(
-            P, circuit_output.verification_key->reference_string->get_precomputed_g2_lines(), 2);
-
         EXPECT_EQ(circuit_output_a.aggregation_state.public_inputs[0].get_value(), inner_inputs[0]);
         EXPECT_EQ(circuit_output_a.aggregation_state.public_inputs[1].get_value(), inner_inputs[1]);
 
-        EXPECT_EQ(inner_proof_result, barretenberg::fq12::one());
-
-        printf("composer gates = %zu\n", outer_composer.get_num_gates());
-
-        bool result = outer_composer.check_circuit();
-        EXPECT_EQ(result, true);
-
-        EXPECT_EQ(check_recursive_proof_public_inputs(
-                      outer_composer,
-                      outer_composer.composer_helper.crs_factory_->get_verifier_crs()->get_precomputed_g2_lines()),
-                  true);
+        check_pairing(circuit_output);
+        check_recursive_verification_circuit(outer_circuit, true);
     }
 
-    // verifies a proof of a circuit that verifies one of two proofs. Test 'a' uses a proof over the first of the two
-    // variable circuits
+    // verifies a proof of a circuit that verifies one of two proofs. Test 'a' uses a proof over the first of the
+    // two variable circuits
```

Comment:
I find that this happens comment code is further commented out then formatted

Reply:
Ah, good call.